Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -245,6 +245,16 @@ private FileSystemMetadata GetFileSystemMetadata(FileSystemInfo info)
                 if (info is FileInfo fileInfo)
                 {
                     result.Length = fileInfo.Length;
+
+                    // Issue #2354 get the size of files behind symbolic links
+                    if(fileInfo.Attributes.HasFlag(FileAttributes.ReparsePoint))
```

Comment:
```suggestion
                    if (fileInfo.Attributes.HasFlag(FileAttributes.ReparsePoint))
```

Current spacing should be throwing a warning

Reply:
Is there a coding standard for Jellyfin?  I didn't see one.

Reply:
We use stylecop in jellyfin-server. It should have given you a warning when compiling or editing the code depending on your IDE settings